test/models/observation_test.rb

```diff
@@ -95,10 +125,23 @@ def test_specimens
     assert(observations(:detailed_unknown).specimens.length > 0)
   end
 
+  def test_observer_takes_general_email_questions_from_viewer
+    user_a = users(:rolf) # rolf takes general email questions
```

Comment:
Since we're using static fixtures, the way to do this is create a separate observation fixture for this test that has a relevant name.  At some point we should consider moving to Factory Girl factories rather than static fixtures.  They essentially let you do what you want to do here without undo database cost.